package.json

```diff
@@ -26,5 +26,8 @@
   "devDependencies": {
     "concurrently": "^6.3.0",
     "husky": "^7.0.4"
+  },
+  "dependencies": {
+    "iso-639-1": "^2.1.15"
```

Comment:
We have two package.json files. One at the root and another one under **./frontend**. I think this one should belong to **./frontend/package.json**. Could you please move it over there?